js/scripts.js

```diff
@@ -0,0 +1,5 @@
+﻿var buttonsArray = document.getElementsByClassName("button");
+
+for(i = 0; i <= buttonsArray.length-1; i++){
```

Comment:
better declare new variable and set in it length of buttons array, because now you are checking length every time when iterator is less or equal than array length